src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+
+import logsParsingAttributesView from 'images/logs_screenshot-crop_parsing-attributes-view.webp'
+
+
+
+
+
+Query time parsing lets you parse your logs directly in the UI without needing to write complex regular expressions or grok patterns. Results are shown instantly since parsing is performed at query time. You can use query time parsing to temporarily extract values from your logs and quickly perform a query on these variables.
+
+## How query time parsing differs from ingest parsing [#differences]
+
+While both types of parsing make it easier for you to query logs, they have some significant differences:
+
+**Ingest parsing:** Parsing during [log ingestion](/docs/logs/ui-data/parsing/) is where you create parsing rules using grok and/or regex. As log records are ingested at New Relic, your parsing rules are applied to create permanent attributes that are stored with your log data in NRDB. These attributes make it easier for you to query log data. 
+
+**Query time parsing:** In contrast to ingest parsing, query time parsing lets you create temporary attributes that will be used as query variables. You can then use these variables in NRQL queries to populate your log table. We automatically create the queries as you make selections in the UI. 
+
+You may choose to use a combination of both parsing approaches. Review the table below to decide if query time parsing is right for you:
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Description 
+            </th>
+            <th>
+                Ingest parsing 
+            </th>
+            <th>
+                Query time parsing 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                Recommended usage
+            </td>
+            <td>
+                Best for creating permanent attributes you can query in the future
+            </td>
+            <td>
+                Best for doing quick queries on attributes that aren't permanent 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Parsing language                
+            </td>
+            <td>
+                You create grok patterns and regular expressions 
+            </td>
+            <td>
+                New Relic creates queries for you using the NRQL [aparse](/docs/nrql/nrql-syntax-clauses-functions/#func-aparse) function 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Timing
+            </td>
+            <td>
+                Applied at ingest
+            </td>
+            <td>
+                Applied when you query
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Results
+            </td>
+            <td>
+                Makes permanent changes to stored log 
+            </td>
+            <td>
+                Temporarily alters the logs table in the UI
```

Comment:
Could we change the phrasing of the alteration to target logs instead of the logs table? I think it would be more precise to say that query time parsing affects the logs themselves. Something similar to this would be great:
`Temporarily alters your logs in the UI`

Reply:
@mchavez-newrelic yes, I will push up this change shortly.